packages/falcon-magento2-api/src/index.test.js

```diff
@@ -131,4 +132,49 @@ describe('Magento2Api', () => {
     // null should not not be converted
     expect(api.convertPathToUrl(null)).toBe(null);
   });
+
+  it('Should not call signIn() after account creation if autoSignIn flag was set to false', async () => {
+    // mock signIn resolver - just return promise that resolves to true
+    api.signIn = sinon.fake.resolves(true);
```

Comment:
@mateuszwozniak please use https://jestjs.io/docs/en/mock-functions#using-a-mock-function

```suggestion
    api.signIn = jest.fn(() => Promise.resolve(true));
```

Reply:
done